US-52 GetById EndPoint

Remma27 opened the pull request:
https://github.com/orgs/utn-integrador-III/projects/7/views/1#

Remma27 committed:
US-52 GetById EndPoint

https://github.com/orgs/utn-integrador-III/projects/7/views/1#

ebarahonautn commented on controllers/zonebyId/controller.py:
@Remma27 @Ale021217 
- en el else del if result: no se debe devolver un NOT_FOUND en el status code porque devolverîa un 404.

En este aspecto debe devolver un OK (200) porque no hubo errores, solamente es que no se encontró la zona indicada, para esto está el message_code NO_DATA para indicar que no hay datos.

lo correcto sería: 
`return ServerResponse(data={}, message="Zone does not exist", message_code=NO_DATA, status=StatusCode.OK)
`
- En el catch de la exception solamente indicar (el metodo ServerResponse devuelve el mensaje de error para ese StatusCode de internal error):

`return ServerResponse(status=StatusCode.INTERNAL_SERVER_ERROR)`

Remma27 committed:
Refactor ZoneByIdController to return proper status codes

https://github.com/orgs/utn-integrador-III/projects/7/views/1?pane=issue&itemId=66296020
